R/proxy.r

```diff
@@ -56,7 +56,7 @@ get_ld_proxies <- function(rsid, bfile, searchspace=NULL, tag_kb=5000, tag_nsnp=
 		return(ld)
 	}
 	ld <- data.table::fread(paste0("gunzip -c ", outname), header=TRUE) %>%
-		dplyr::as_tibble() %>%
+		dplyr::as_tibble(.data, .name_repair="minimal") %>%
```

Comment:
Recommend deleting the `.data, ` as it's not needed (and I think this is what's causing the error in building guide.Rmd).